schemas/json/README.md

```diff
@@ -95,18 +104,17 @@ First, a [JSON number] can only be a concrete numerical value where special case
 Second, the [lexical space] of XSD values is much larger than the lexical space of JSON numbers and booleans.
 For example, both `1` and `true` are valid lexical representations of a true value.
 In contrast, JSON recognizes only `true`.
+We are not considering cases in which a semantic definition would even redefine the semantics of `1` as `false`.
```

Comment:
This is not possible by XSD. XSD is very explicit on both lexical and canonical form. The canonical form of "1" is "True" IIRC.

Reply:
You were giving this example originally ....
I will just remove the sentence. Your sentence:

"For example, we might define through the attribute [Property/semanticId](https://www.plattform-i40.de/IP/Redaktion/DE/Downloads/Publikation/Details_of_the_Asset_Administration_Shell_Part1_V3.pdf?__blob=publicationFile&v=10#page=75) that a [Property/value](https://www.plattform-i40.de/IP/Redaktion/DE/Downloads/Publikation/Details_of_the_Asset_Administration_Shell_Part1_V3.pdf?__blob=publicationFile&v=10#page=75), given as a boolean, distinguishes between 1 and true where 1 has a completely different meaning from true. This distinction would have been lost if we simply serialized the value to a JSON boolean."